tf-show-parser/package.json

```diff
@@ -3,7 +3,7 @@
     "prepare": "ncc build main.js -o ../dist --source-map --license licenses.txt"
   },
   "dependencies": {
-    "@actions/core": "^1.8.2"
+    "@actions/core": "^1.10.0"
```

Comment:
update actions/core to a version that setup the outputs with the new approach `>> $GITHUB_OUTPUT`